### ISSUE TITLE
Substitute 0777 with 0o777

### ISSUE THREAD

**Issue by dilyanpalauzov** [asciidoc-py/asciidoc-py#250]
Python 3.10 prints with asciidoc 10.1.4:
```
Traceback (most recent call last):
  File "/usr/local/bin/asciidoc", line 5, in <module>
    from asciidoc.asciidoc import cli
  File "/usr/local/bin/asciidoc.py", line 5685
    perms = (zi.external_attr >> 16) & 0777
                                       ^
SyntaxError: leading zeros in decimal integer literals are not permitted; use an 0o prefix for octal integers
make: *** [Makefile:41: check] Error 1
```
Please replace `0777` with `0o777`.

**Comment by dilyanpalauzov** [comment]
That file is from some other, older asciidoctor interpretation.

**Comment by MasterOdin** [comment]
Yup, this was fixed in the 9.0.0 release where 2to3 was run over the old python2 codebase: 3c0659f4e8ce8d5b49b505dc36e05d4e30796b32
